manager/backup.go

```diff
@@ -55,6 +56,10 @@ func SyncVolumeLastBackupWithBackupVolume(volumeName string, backupVolume *engin
 			}
 			return err
 		}
+		// do not update standby volume here
```

Comment:
Why?

Reply:
Since standby volume name is not the same as its backup volume name, backupVolume is nil here. Then LastBackup of standby volume will be empty.